codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPProtocolCustomizations.kt

```diff
@@ -42,8 +43,7 @@ abstract class AWSHTTPProtocolCustomizations : DefaultHTTPProtocolCustomizations
         op: OperationShape
     ) {
         if (op.isInputEventStream(ctx.model) && op.isOutputEventStream(ctx.model)) {
-            writer.addImport(AWSSwiftDependency.AWS_SDK_EVENT_STREAMS_AUTH.target)
```

Comment:
Importing targets directly is now only done in test files.  I won't comment on this in following files.